azure-pipelines-azurezagent/files/start.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+export AGENT_ALLOW_RUNASROOT="1"
+
+print_header() {
+  lightcyan='\033[1;36m'
+  nocolor='\033[0m'
+  echo -e "${lightcyan}$1${nocolor}"
+}
+
+source ./env.sh
+
+print_header "Running Azure Pipelines agent..."
+
+trap 'exit 0' EXIT
+trap 'exit 130' INT
+trap 'exit 143' TERM
+
+# copy the config files and credentials in workdir
+cp /mnt/config/.agent ./
+cp /mnt/config/.credentials ./
+cp /mnt/config/.credentials_rsaparams ./
```

Comment:
don't we need to make sure the permissions on these are also correct? Either let quattor set them and use `cp -a` or fix it here?

Reply:
I tried 'cp -ap' in the Dockerfile, but I got error messages... still have to understand why...